app/ante/ante.go

```diff
@@ -56,52 +88,54 @@ func NewAdjustGasPriceDecorator() AdjustGasPriceDecorator {
 // AnteHandle adjusts the gas price based on the tx type.
 func (r AdjustGasPriceDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
 	msgs := tx.GetMsgs()
-	if len(msgs) == 1 && (strings.Contains(strings.ToLower(sdk.MsgTypeURL(msgs[0])), strings.ToLower(disptypes.MsgTypeCreateDistribution)) ||
-		strings.Contains(strings.ToLower(sdk.MsgTypeURL(msgs[0])), strings.ToLower(disptypes.MsgTypeRunDistribution))) {
+	if len(msgs) == 1 && slicex.StringsContain(strings.ToLower(sdk.MsgTypeURL(msgs[0])), distributionMessageTypes) {
```

Comment:
How about something like that
`if len(msgs) == 1 && isDistributedMsg(msgs[0])`
where
`func isDistributedMsg(msg sdk.Msg) bool {
     return slicex.StringsContain(strings.ToLower(sdk.MsgTypeURL(msg)), distributionMessageTypes)
}`

Sorry for the terrible formatting

Reply:
Good point, done